OpenEdXMobile/src/main/java/org/edx/mobile/interfaces/OnActivityResultListener.java

```diff
@@ -21,10 +21,10 @@
      * didn't return any result, or crashed during its operation.
      * <p>You will receive this call immediately before onResume() when your
      * activity is re-starting.</p>
-     * <p>
      * <p>This method is never invoked if your activity sets
      * {@link android.R.styleable#AndroidManifestActivity_noHistory noHistory} to
-     * <code>true</code>.
+     * <code>true</code>. </p>
```

Comment:
You should also surround the first paragraph with the `<p>` tag, to be consistent.